decktape.js

```diff
@@ -51,6 +51,13 @@ parser.script('decktape').options({
     default : 0,
     help    : 'Duration in milliseconds between the page has loaded and starting to export slides',
   },
+  enableSubsteps: {
```

Comment:
This would basically cover #24. I wonder what could be the best terminology across all the HTML presentation frameworks. Reveal calls it _fragments_, other refer to _incremental_ display of slide content...